### PR TITLE
update(package.json): updated standard version

### PR DESCRIPTION
**Pull request by zoepage**

### CODE REVIEW

**Comment by gr2m** [comment]
can you please reword the commit to `chore(package): standard@3.11.1`?

File: package.json [hunk 1]
```diff
@@ -6,13 +6,14 @@
     "url": "https://github.com/gr2m/pouchdb-hoodie-sync/issues"
   },
   "dependencies": {
-    "pouchdb-extend": "^0.1.2"
+    "pouchdb-extend": "^0.1.2",
+    "standard": "^3.11.1"
```
Comment (gr2m) [inline package.json]: please don't add `standard` to `dependencies` :)

**Comment by zoepage** [comment]
done :)

**Comment by gr2m** [comment]
merged via 3a9a30c :rocket: :guitar:

**Pull request closed without merging**